otter/integration/lib/autoscale.py

```diff
@@ -194,10 +204,10 @@ def wait_for_N_servers(
             Otherwise, an exception is raised.
         """
 
-        def check(state):
-            if state[0] != 200:
+        def check((code, response)):
```

Comment:
Tuple parameter unpacking is deprecated, and doesn't work in 3.x+; see [PEP 3113](http://legacy.python.org/dev/peps/pep-3113/). Suggestion: keep the "state" parameter, but destructure on the next line:

```python
def check(state):
    code, response = state
    ....
```